packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -449,16 +449,30 @@ function buildPrivateFieldInitLoose(ref, prop, privateNamesMap) {
   `;
 }
 
-function buildPrivateInstanceFieldInitSpec(ref, prop, privateNamesMap) {
+function buildPrivateInstanceFieldInitSpec(ref, prop, privateNamesMap, state) {
   const { id } = privateNamesMap.get(prop.node.key.id.name);
   const value = prop.node.value || prop.scope.buildUndefinedNode();
 
-  return template.statement.ast`${t.cloneNode(id)}.set(${ref}, {
-    // configurable is always false for private elements
-    // enumerable is always false for private elements
-    writable: true,
-    value: ${value},
-  })`;
+  if (!state.availableHelper("classPrivateFieldInitSpec")) {
+    return template.statement.ast`${t.cloneNode(id)}.set(${ref}, {
+      // configurable is always false for private elements
+      // enumerable is always false for private elements
+      writable: true,
+      value: ${value},
+    })`;
+  }
```

Comment:
Since this is self-contained and it won't be needed in Babel 8, we could wrap it with a `process.env.BABEL_8_BREAKING` guard so that we can automatically remove it when bumping our major version:

```suggestion
  if (!process.env.BABEL_8_BREAKING) {
    if (!state.availableHelper("classPrivateFieldInitSpec")) {
      return template.statement.ast`${t.cloneNode(id)}.set(${ref}, {
        // configurable is always false for private elements
        // enumerable is always false for private elements
        writable: true,
        value: ${value},
      })`;
    }
  }
```

Reply:
Done!